vendor/assets/javascripts/modernizr.js

```diff
@@ -0,0 +1,1406 @@
+/*!
```

Comment:
Can we load this (preferably an uglified version) from another CDN? This is half of your PR and isn't actually our code.